### PR TITLE
Fix/tao 7087/actor details missing when use kvstorage

### PR DESCRIPTION
**Pull request by SashaTAO**

### CODE REVIEW

File: models/classes/user/LtiUserHelper.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+use oat\generis\model\GenerisRdf;
+use oat\oatbox\service\ConfigurableService;
+use oat\oatbox\service\exception\InvalidService;
+
+/**
+ * Class LtiUserHelper
+ * @package oat\taoLti\models\classes\user
+ */
+class LtiUserHelper extends ConfigurableService
```
Comment (zagovorichev) [inline models/classes/user/LtiUserHelper.php]: The name of the `service` is `Helper`, it's misleading

File: views/build/.htaccess [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<IfModule mod_rewrite.c>
```
Comment (zagovorichev) [inline views/build/.htaccess]: I don't think that `.htaccess` files are necessary for the task

File: models/classes/user/LtiUserHelper.php (hunk 1 above)
Comment (zagovorichev) [reply models/classes/user/LtiUserHelper.php]: tbh I don't think that we need this helper at all, why can't we just use `LtiUserService`?

File: views/build/.htaccess (hunk 2 above)
Comment (SashaTAO) [reply views/build/.htaccess]: Yes, it was generated automatically by mistake

**Pull request merged**